### DISCUSSION TITLE
Button doesn't use relative units

### DISCUSSION THREAD

**Discussion by wolkyura** [heroui-inc/heroui#1685] [General]
I noticed that `Button` uses pixels to define its dimensions (for `height`, `padding`, etc.). I'm wondering if this is intentional, almost all components use relative units (`rem`), which works fantastic when trying to make a layout more responsive on mobile devices, however, the `Button` is the one that stays the same size and looks weird on compared to the rest.

As a workaround I could change the size of the button on smaller screens, however, I'm curious if there's a specific reason or limitation behind this design choice and would appreciate any insights or suggestions.

**Comment by jrgarciadev** [comment]
https://github.com/nextui-org/nextui/pull/1688
